apps/web/src/components/editor/tiptap.tsx

```diff
@@ -443,14 +443,43 @@ function TiptapWrapper(
     }
   }, [isHydrating]);
 
+  useEffect(() => {
```

Comment:
We should be doing this in `EditorChrome` in `components/editor/index.tsx`.

Reply:
`editorContainerRef` in `EditorChrome` component is the entire editor, so the zoom property gets applied to the properties bar and the note title as well. I've kept this `useEffect` in `tiptap` for now, please let me know if you have any suggestion

Reply:
Okay, that makes sense but is there a way to not do this inside a useEffect and directly use the `onWheel` event on the `Flex` below?

Reply:
I get these errors in the browser console when I use the `onWheel` prop on `Flex`: 

```
Unable to preventDefault inside passive event listener invocation.
```

Currently investigating them...

Although, I've removed the unnecessary `editorConfig.zoom` dependency in the `useEffect`.

Reply:
Got it. I think React uses passive event listeners by default so we are forced to use refs.